// weebs-wrapper/index.js
exports.coin = getCoin();
exports.hug = getHug();
exports.kiss = getKiss();
exports.laugh = getLaugh();
exports.pat = getPat();
exports.poke = getPoke();
exports.punch = getPunch();
exports.slap = getSlap();

function getCoin() {
  var path = 'https://img.weebs.cf/img/coin/';
  var fileType = '.gif';
  var rand = Math.floor(Math.random() * 2) + 1;
  return path + rand + fileType;
}

function getHug() {
  var path = 'https://img.weebs.cf/img/hug/';
  var fileType = '.gif';
  var rand = Math.floor(Math.random() * 19) + 1;
  return path + rand + fileType;
}

function getKiss() {
  var path = 'https://img.weebs.cf/img/kiss/';
  var fileType = '.gif';
  var rand = Math.floor(Math.random() * 20) + 1;
  return path + rand + fileType;
}

function getLaugh() {
  var path = 'https://img.weebs.cf/img/laugh/';
  var fileType = '.gif';
  var rand = Math.floor(Math.random() * 21) + 1;
  return path + rand + fileType;
}

function getPat() {
  var path = 'https://img.weebs.cf/img/pat/';
  var fileType = '.gif';
  var rand = Math.floor(Math.random() * 21) + 1;
  return path + rand + fileType;
}

function getPoke() {
  var path = 'https://img.weebs.cf/img/poke/';
  var fileType = '.gif';
  var rand = Math.floor(Math.random() * 20) + 1;
  return path + rand + fileType;
}

function getPunch() {
  var path = 'https://img.weebs.cf/img/punch/';
  var fileType = '.gif';
  var rand = Math.floor(Math.random() * 20) + 1;
  return path + rand + fileType;
}

function getSlap() {
  var path = 'https://img.weebs.cf/img/slap/';
  var fileType = '.gif';
  var rand = Math.floor(Math.random() * 21) + 1;
  return path + rand + fileType;
}
